projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -160,6 +160,12 @@ public static function load_wpcom_user_features() {
 		require_once __DIR__ . '/features/wpcom-sidebar-notice/wpcom-sidebar-notice.php';
 		require_once __DIR__ . '/features/wpcom-themes/wpcom-themes.php';
 
+		// We include WPCom Themes results and installation on WoA sites only and non-WP_CLI context. This is because we want to retain the core themes behaviour in the CLI context.
+		// For now we don't want this to run on Simple sites, although the plan is to use this code to match the Simple and Atomic experience using these classes.
+		if ( ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) && ( ! defined( 'WP_CLI' ) || ! WP_CLI ) ) {
+			require_once __DIR__ . '/features/wpcom-themes/wpcomsh-themes.php';
+		}
```

Comment:
What is the plan for removing this atomic gate? Can that be done here, or as part of a follow-up PR? Asking because I think it will help with a PR for the atomic transfer process (https://github.com/Automattic/wp-calypso/pull/99761). And technically this shouldn't have any effect on simple sites yet anyway, since they don't call any install code.

Reply:
Ah, I see the PR description says "This matches existing Production but sets us up for future development", so I assume a future PR.